### ISSUE TITLE
Pushed gems disappearing from specs

### ISSUE THREAD

**Issue by janbiedermann** [rubygems/rubygems.org#2191]
Dec 15 i pushed several gems of the isomorfeus https://isomorfeus.com/ project, version 1.0.0.zeta6.
I received email confirming the push.
They appear in the webinterface but not in the specs, even older versions where removed from the specs.
The pushed gems also do not appear in the dashboard of my rubygems profile.

Example:
https://rubygems.org/gems/isomorfeus-data

Lists latest version as 1.0.0.zeta6

gem sources -u
gem search -r isomorfeus-data -> nothing

all previous version disappeared from the specs:

gem search isomorfeus

*** REMOTE GEMS ***

isomorfeus-puppetmaster (0.3.0)
isomorfeus-react (16.12.1)
isomorfeus-redux (4.0.16)
isomorfeus-speednode (0.2.11)

⬆️ 
No isomorfeus-data, also all other isomorfeus gems (1.0.0.zeta6) pushed yesterday disappeared.

Thats very annoying for other projects depending on isomorfeus.
Please help.

**Comment by dwradcliffe** [comment]
I took a look and here's what I found:
- Your gems install fine (`gem install isomorfeus-data -v 1.0.0.zeta8`)
- I can see your gems and all the versions on rubygems.org
- I can search for your gems on rubygems.org

The only thing that doesn't work is `gem search`. I'm guessing that's somehow because you only have pre-release versions. It must be filtering those? Maybe @sonalkr132 can confirm.

This really shouldn't prevent anything from working though. If I've missed something let me know.
